docs/pages/divider.js

```diff
@@ -1,64 +1,309 @@
 // @flow strict
 import type { Node } from 'react';
-import PropTable from '../components/PropTable.js';
 import PageHeader from '../components/PageHeader.js';
-import CardPage from '../components/CardPage.js';
 import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
 
-const cards: Array<Node> = [];
-const card = (c) => cards.push(c);
+export default function DividerPage(): Node {
+  return (
+    <Page title="Divider">
+      <PageHeader
+        name="Divider"
+        description="If you have two things that need to be separated, put a `Divider` between them."
```

Comment:
- "things" is a bit generic…perhaps "elements"?
- No need for the backticks around Divider — we've moved away from that style for component names

Reply:
DONE